### PR TITLE
Update windows installation instructions in README

### PR DESCRIPTION
**Pull request by chick**
Update windows install section based on user feedback.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,20 +99,65 @@ jupyter notebook
...
 ```
 java -noverify -jar coursier launch -r sonatype:releases -r sonatype:snapshots -i ammonite -I ammonite:org.jupyter-scala:ammonite-runtime_2.11.11:0.8.3-1 -I ammonite:org.jupyter-scala:scala-api_2.11.11:0.4.2 org.jupyter-scala:scala-cli_2.11.11:0.4.2 -- --id scala --name "Scala"
 ```
 
-Download this repository as a zip file (or use a Windows git client) and unpack it.
-Install the customization script by moving `generator-bootcamp/source/custom.js` to `~/.jupyter/custom/custom.js`.
+#### Install the generator-bootcamp repo.
+Download the [generator-bootcamp](https://github.com/ucb-bar/generator-bootcamp) as a zip file (or use a Windows git client)
+and unpack it.
+I copied it to my c:\Users\me\generator-bootcamp.
```
Comment (edwardcwang) [inline README.md]: This may be overly specific

File: README.md [hunk 2]
```diff
@@ -99,20 +99,65 @@ jupyter notebook
...
+
+#### Install Jupyter
+Jupyter recommends using the Anaconda distribution, here is the 
+[Windows download](https://www.anaconda.com/download/#windows).
+
+Near the end of the Jupyter installation is a question about whether to add Jupyter to the PATH.
+Windows does not recommend this, but I do.  It will make it easier to run using the command prompt.
+ 
+#### Install Scala components.
+The simplest way seens to be to Download [Coursier from https://git.io/vgvpD](https://git.io/vgvpD)
```
Comment (edwardcwang) [inline README.md]: Would move "Coursier from" out of the link text

File: README.md (hunk 1 above)
Comment (chick) [reply README.md]: Probably, but I'd prefer to keep based on my own experiences going through this procedure.

File: README.md (hunk 2 above)
Comment (chick) [reply README.md]: Will do

**Pull request merged**